node/full_test.go

```diff
@@ -14,7 +14,7 @@ import (
 
 func TestNewFull(t *testing.T) {
 	repo := MockRepository(t, DefaultFullConfig())
-	node, err := NewFull(repo)
+	node, err := New(Full, repo)
```

Comment:
So, now we can either 2 options with `New(Full, repo)` and `NewFull(repo)`, right?
Can we leave a recommendation which is more preferable?

Reply:
Yeah, we should document which to run in which circumstances. But that is less on the code level but more on the CLI directly IMO.